src/odemis/gui/xmlh/resources/panel_tab_fastem_main.xrc

```diff
@@ -0,0 +1,140 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<resource class="wxFrame" version="2.5.3.0"
+  xmlns="http://www.wxwidgets.org/wxxrc">
+  <object class="wxPanel" name="pnl_tab_fastem_main">
+    <object class="wxBoxSizer">
+      <object class="sizeritem">
+        <object class="wxPanel" name="pnl_user_settings">
+          <flag>wxEXPAND</flag>
+          <bg>#333333</bg>
+          <size>300,600</size>
+        </object>
+      </object>
+      <orient>wxHORIZONTAL</orient>
+      <object class="sizeritem">
+        <object class="wxPanel" name="pnl_toolbar">
+          <object class="wxBoxSizer">
+            <orient>wxVERTICAL</orient>
+            <object class="spacer">
+              <size>-1,20</size>
+            </object>
+            <object class="sizeritem">
+              <object class="ImageButton" name="btn_pnl_user_settings">
+                <icon>../../img/icon/ico_chevron_up.png</icon>
```

Comment:
Could you add icons for the left/right version. up/down doesn't fit here.

Reply:
I'm now thinking you could even try to avoid creating a PNG by using a standard button and switching between `≪` and `≫`  characters.